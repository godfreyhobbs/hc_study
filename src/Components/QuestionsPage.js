import React, {Component} from 'react';
import IframeQuestions from './IframeQuestions';
import '../App.css';

class QuestionsPage extends Component {

  handleClick() {
    this.props.callback("Payment");
  }

  render() {
    return (
       <div>
         <button onClick={(e) => this.handleClick(e)}>
           Get Your Money!
         </button>
         <IframeQuestions></IframeQuestions>
       </div>
    );
  }
}

export default QuestionsPage;
