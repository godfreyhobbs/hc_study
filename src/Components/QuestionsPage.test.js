import React from 'react';
import ReactDOM from 'react-dom';
import QuestionsPage from './QuestionsPage';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<QuestionsPage />, div);
  ReactDOM.unmountComponentAtNode(div);
});



it('renders correctly', () => {
  const tree = renderer
.create(<QuestionsPage />)
   .toJSON();
expect(tree).toMatchSnapshot();
});
