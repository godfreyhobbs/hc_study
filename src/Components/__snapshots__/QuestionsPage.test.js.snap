// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h1
      className="App-title"
    >
      Welcome to the Viel Cornwell Hookah Study
    </h1>
  </header>
  <div>
    Insert Google Form here
  </div>
  <button
    onClick={[Function]}
  >
    Submit
  </button>
</div>
`;
